### PR TITLE
Synced prose with schema (CWE changes), added refs, and JSON base security considerations

### PR DESCRIPTION
**Pull request by sthagen**
As input to the upcoming CSAF TC September meeting some small enhancements:

* Synced prose with schema (CWE changes), 
* added JSON and JSON text sequences refs, 
* added JSON base security blurb, 
* updated dates and fixed nits

@tschmidtb51 this is the sync I avisaged in #135

### CODE REVIEW

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 1]
```diff
@@ -302,6 +304,8 @@ _Information technology — Security techniques — Vulnerability disclosure_, I
 https://www.iso.org/standard/45170.html.
 ###### [RFC3552]
 Rescorla, E. and B. Korver, "Guidelines for Writing RFC Text on Security Considerations", BCP 72, RFC 3552, DOI 10.17487/RFC3552, July 2003, https://www.rfc-editor.org/info/rfc3552.
+###### [RFC7464]
+N. Williams., "JavaScript Object Notation (JSON) Text Sequences", RFC 8259, DOI 10.17487/RFC7464, February 2015, http://www.rfc-editor.org/info/rfc7464.
```
Comment (sthagen) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:308]: Based on this added reference we can add best practice advice how to stream or batch CSAF documents in high volume or near real time websocket/long polling/push scenarios (i.e. across long living connections)

**Comment by santosomar** [comment]
This is pretty straightforward and I will merge now and show the markdown during the TC meeting. We can obtain feedback and members can open any new issues if any changes are needed. I just want to have a clean document in the master branch for everyone to review.

**Comment by santosomar** [comment]
Thank you so much for all your time and help!

**Comment by santosomar** [comment]
Thank you so much for all your time and help!

**Pull request merged**